content/pt/docs/concepts/resources/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Resources
+weight: 70
+default_lang_commit: a1740fd934e595f1396f2eb82a58a80824369b09
+---
+
+## Introdução
+
+{{% docs/languages/resources-intro %}}
+
+Se você usar [Jaeger](https://www.jaegertracing.io/) como seu backend de
+observabilidade, os _resource attributes_ serão agrupados na guia **Process**:
```

Comment:
Why not translating resource attributes?

Reply:
I think the consensus was to avoid translating terms that are used by OpenTelemetry components. "Resource attributes" are mentioned in places such as the env var `OTEL_RESOURCE_ATTRIBUTES`, so translating the term could disconnect it from its use.

But I'm actually not sure if "resource attributes" specifically was discussed. @edsoncelio @EzzioMoreira do you know of any prior discussions about it?

Reply:
We decided to translate terms like `metric` or `trace` in the Spanish localization. See Microsoft docs for a translation of resource attributes: https://learn.microsoft.com/pt-br/azure/azure-monitor/app/opentelemetry-configuration?tabs=aspnetcore#set-the-cloud-role-name-and-the-cloud-role-instance

Reply:
My 10cents here:

We are translating terms like metrics or traces, too, @theletterf; we have a document pinned on our Slack channel with the list of terms that we translate or do not base on some discussions. This case is different, in particular, because "recurso" in Portuguese has many meanings, so we are careful about whether or not to translate it.

In this case, I agree with translating something like:
"os atributos dos recursos são agrupados ...."

Sounds good to me because we're talking about attributes (atributos) generated from resources (recursos).

I was using this doc as a reference:  https://opentelemetry.io/docs/specs/otel/resource/sdk/

But we can open a thread on Slack to discuss this further if you all agree!

Reply:
> We decided to translate terms like `metric` or `trace` in the Spanish localization. See Microsoft docs for a translation of resource attributes: https://learn.microsoft.com/pt-br/azure/azure-monitor/app/opentelemetry-configuration?tabs=aspnetcore#set-the-cloud-role-name-and-the-cloud-role-instance

Oh cool, thanks for the reference! I would 👍 translating "resource attributes"